frame/ddc-metrics-offchain-worker/src/tests/test_runtime.rs

```diff
@@ -167,6 +167,7 @@ parameter_types! {
     pub const SurchargeReward: Balance = 150;
     pub const MaxDepth: u32 = 100;
     pub const MaxValueSize: u32 = 16_384;
+    pub MaxCodeSize: u32 = 256 * 1024;
```

Comment:
This value is higher than we have in node runtime. Probably we need to increase it there to be able to upload a SC